app/models/drawing_wizard/archive_form.rb

```diff
@@ -6,8 +6,8 @@ class ArchiveForm < BaseForm
     validates :updated_at, presence: true
 
     validates :archive_reason,
-              inclusion: {in: %w(scale design dimensions other),
+              inclusion: { in: %w(scale design dimensions other),
```

Comment:
We might be able to avoid some duplication here. We should be able to ask the `Drawing` for its list of reasons. E.g. `Drawing.archived_reasons.keys`